src/applications/personalization/dashboard/reducers/prescriptions.js

```diff
@@ -69,7 +68,7 @@ export default function prescriptions(state = initialState, action) {
         };
       }
 
-      return assign(state, newState);
+      return { ...state, ...newState };
```

Comment:
One of these days I'm going to try using Immer in our reducers like the Redux maintainers have been recommending for ages.